commands/removeCardCommand.js

```diff
@@ -1,36 +1,77 @@
 import db from "../db.js";
 
-const removeCardCommand = (message) => {
+const removeCardCommand = async (message) => {
   const cardToRemove = message.content.split(" ").slice(1).join(" ");
   const ref = db.ref("users");
-  ref.once("value", (snapshot) => {
-    const userData = snapshot.val();
-    const userID = message.author.id;
-    const userInventory = userData[userID].inventory;
-
-    const cardIDToRemove = Object.keys(userInventory).find(
-      (key) =>
-        userInventory[key].name.toLowerCase() === cardToRemove.toLowerCase()
+  const snapshot = await ref.once("value");
+  const userData = snapshot.val();
+  const userID = message.author.id;
+  const userInventory = userData[userID].inventory;
+
+  const cardIDToRemove = Object.keys(userInventory).find(
+    (key) =>
+      userInventory[key].name.toLowerCase() === cardToRemove.toLowerCase()
+  );
+
+  if (cardIDToRemove) {
+    const deletedCardObject = userInventory[cardIDToRemove];
```

Comment:
by convention we can assume that most variables with a noun name are objects, unless it's clearly another type (like a `name` is string). So if you just name this `deletedCard`, you can always assume its a Card object.